src/main.go

```diff
@@ -42,15 +43,16 @@ var (
 	// instance is composed of ping, jitter, flood-ping and monitor
 	// chain paths. matrix is used in the monitoring screen to
 	// reduce the http request by grouping them based on routes.
-	// Without matrix, the http traffic would increase 4 times
-	// the current count.
+	// Filters out identical routes that arrive to be monitored.
 	matrix   = make(map[string]*utils.BRMatrix)
 	reload   = make(chan struct{})
 	done     = make(chan struct{})
 	conf     *parser.Config
 	chainSet = tsdb.NewChainSet(tsdb.FlushAsTime, time.Second*300)
 	// targetMachineCalc contains calculations that are machine/vm/load-balancer
 	// specific. These involve use of IP addresses/Domain names respectively.
+	// We use this to keep track of host names because all the routes that share
+	// the same hostname will have the same ping, jitter and flood-ping.
```

Comment:
This is a good replacement of comment, but I do feel the 45 & 46 line comment should be added here. It is useful for contributors to know why this behaviour was done. Helps in understanding the already complex map.